independent-projects/resteasy-reactive/server/processor/src/main/java/org/jboss/resteasy/reactive/server/processor/generation/exceptionmappers/ServerExceptionMapperGenerator.java

```diff
@@ -645,9 +646,8 @@ private static void checkModifiers(MethodInfo info) {
     }
 
     private static String getGeneratedClassName(MethodInfo targetMethod, Type handledExceptionType) {
-        return targetMethod.declaringClass().name() + "$GeneratedExceptionHandlerFor$"
-                + handledExceptionType.name().withoutPackagePrefix()
-                + "$OfMethod$" + targetMethod.name();
+        return targetMethod.declaringClass().name() + "$ExceptionMapper$"
```

Comment:
Why not replace `$ExceptionMapper$` with the method name?

Reply:
I find that looking at this name I know for sure that is a generated exception mapper without having to look into the code itself

Reply:
Indeed you know what it is, but you don't know where it's defined, so you can't find it back easily.

Reply:
Whereas if you can find it back, then it's pretty obvious what it is once you look at it.